app/src/main/java/org/p2p/wallet/swap/jupiter/statemanager/token_selector/SwapInitialTokenSelector.kt

```diff
@@ -0,0 +1,37 @@
+package org.p2p.wallet.swap.jupiter.statemanager.token_selector
+
+import org.p2p.core.token.Token
+import org.p2p.core.utils.Constants
+import org.p2p.wallet.swap.jupiter.domain.model.SwapTokenModel
+import org.p2p.wallet.swap.jupiter.repository.model.JupiterSwapToken
+
+interface SwapInitialTokenSelector {
+
+    suspend fun getTokenPair(): Pair<SwapTokenModel, SwapTokenModel>
+}
+
+fun jupiterSwapGetTokenB(
+    jupiterTokens: List<JupiterSwapToken>,
+    userTokens: List<Token.Active>,
+    findSolOrUSDC: Boolean,
```

Comment:
findSolOrUsdc

Reply:
or maybe even just preferSol?
because that's what you do if the flag is true - you put sol or else usdc